### PR TITLE
Find profile filters first by berth data and then by profile data

### PR DESCRIPTION
**Pull request by nikomakela**
VEN-1469. Filter first by berth filters, then by profile filters and then return the berth query in the right order.

### CODE REVIEW

File: customers/services/profile.py [hunk 1]
```diff
@@ -367,7 +374,7 @@ def create_profile(
         response = self.query(query=mutation, variables=variables)
         global_id = response.get("createProfile", {}).get("profile", {}).get("id")
         profile_id = UUID(from_global_id(global_id))
-        profile = CustomerProfile.objects.create(id=profile_id)
+        profile = CustomerProfile.objects.get(id=profile_id)
```
Comment (charn) [inline customers/services/profile.py:377]: What's this change?

Comment (nikomakela) [reply customers/services/profile.py:377]: Good catch! It's an accident! I used the part of the script to import local data

File: customers/schema/queries.py [hunk 2]
```diff
@@ -202,56 +209,62 @@ class Query:
...
-        marked_winter_storage_areas = kwargs.pop("marked_winter_storage_areas", [])
-        marked_winter_storage_places = kwargs.pop("marked_winter_storage_places", [])
-
-        unmarked_winter_storage_lease_customer = kwargs.pop(
-            "unmarked_winter_storage_lease_customer", False
-        )
-        unmarked_winter_storage_areas = kwargs.pop("unmarked_winter_storage_areas", [])
+        harbors = [h for h in kwargs.pop("harbors", []) if h is not None]
+        piers = [p for p in kwargs.pop("piers", []) if p is not None]
+        berths = [b for b in kwargs.pop("berths", []) if b is not None]
```
Comment (charn) [inline customers/schema/queries.py:214]: Added a comment related to this on the other PR https://github.com/City-of-Helsinki/berth-reservations/pull/660#pullrequestreview-889545260

**Review by charn — APPROVED** [review]
LGTM! :+1: :shipit: That one remaining comment I had is not really a show stopper.

File: customers/schema/queries.py (hunk 2 above)
Comment (nikomakela) [reply customers/schema/queries.py:214]: Trivial / minor in my opinion. It would need some changes to UI also. Lets do it later if decided

**Pull request merged**